corehq/apps/style/templates/style/bootstrap2/base.html

```diff
@@ -30,7 +30,12 @@
         {% else %}
         <script src="{% static 'jquery-1.7.1-legacy/jquery.js' %}"></script>
         {% endif %}
+
+        {% if upgrade_knockout_js %}
+        <script type="text/javascript" src="{% new_static 'knockout/dist/knockout.js' %}"></script>
+        {% else %}
         <script type="text/javascript" src="{% static 'knockout-2.3.0-legacy/knockout.js' %}"></script>
```

Comment:
no, this is the `base.html` so any page without `upgrade_knockout_js` set to True won't have it loaded at all